charts/incubator/alfresco-dbp/values.yaml

```diff
@@ -185,7 +183,6 @@ alfresco-content-services:
         -Dalfresco.cluster.enabled=true
         -Ddeployment.method=HELM_CHART
         -Xms2000M -Xmx2000M
-        -Ddsync.service.uris=\"$SYNC_SERVICE_URI\"
```

Comment:
We'll still need to supply this value.

Reply:
See the comment above.

Reply:
We still need to set the URI based on the domain being used for ingress into the cluster.

Reply:
I understand, makes perfect sense. I reverted the change.